src/index.js

```diff
@@ -1,2 +1,32 @@
+/* eslint-disable import/no-cycle */
+/* eslint-disable no-unused-vars */
+/* eslint-disable import/no-mutable-exports */
+/* eslint-disable no-plusplus */
+// eslint-disable-next-line no-unused-vars
 import _ from 'lodash';
 import './style.css';
+
+const Tasks = document.querySelector('.tasks');
+const Input = document.querySelector('form input');
+const AddBtn = document.querySelector('form button');
+const Span = document.querySelector('span');
+
+// eslint-disable-next-line import/prefer-default-export, prefer-const
+let tasks = [
+  { index: 0, completed: false, description: 'Wash dishes' },
+  { index: 0, completed: false, description: 'Wash dishes' },
+];
```

Comment:
You have done great work on this project 👏 However, the task objects inside your tasks array here, all have the same value for their index keys. I think you should change this and perhaps set the index keys to start from 1 instead of 0. You should also ensure that they appear on the page according to the order of their index values. This was part of the project requirement: 

![image](https://user-images.githubusercontent.com/107795045/221261438-a53d4c7f-40eb-467e-9894-f43b1391f2e3.png)

Your tasks array should end up looking something like this: 
```
let tasks = [
  { index: 1, completed: false, description: 'Wash dishes' },
  { index: 2, completed: false, description: 'Wash dishes' },
];
```